docs/index.md

```diff
@@ -149,6 +149,37 @@ Then make sure that GitHub Pages is enabled in
 If all went well, your website should be available soon via
 `https://<repo-owner>.github.io/<repo-name>/`.
 
+#### Website versions using Mike
+
+The above commands create a new documentation version using
+[Mike](https://pypi.org/project/mike/), which is used to keep multiple versions
+of the website.
+
+The new documentation version is called `next`, which is used as the name for
+the currently in-development branch. The `next` branch has an alias called
+`latest`, which is set as the *default*.
+
+If the website is visited without specifying an explicit version, the `latest`
+version will be displayed. It is recommended to point `latest` to the latest
+stable version instead of the currently in-devepoment version, so as you as you
```

Comment:
duplicated as you as you

Sorry I've just spotted it before

Reply:
Actually `s/you/soon/`: "as soon as you" :)

Reply:
Fixed.